src/components/reviews.tsx

```diff
@@ -26,32 +26,24 @@ const useStyles = makeStyles((theme: Theme) =>
 
 export const Reviews: React.FC<{ id: string }> = (props) => {
   const classes = useStyles();
-  const [reviews, setRewiews] = React.useState<Review[]>([]);
+  const [reviews, setReviews] = React.useState<Review[]>();
   const { id } = props;
-
-  if (!id) {
-    return null;
-  }
-
-  console.log(reviews, id);
-
-  React.useEffect(() => {
+  if(!reviews){
     fetch(EndPoint.reviews(id))
```

Comment:
as we are not doing server side rendering currently, so I believe network request should be within `useEffect`.
i.e after initial rendering of the component.